reports/management/commands/generate_dataset.py

```diff
@@ -0,0 +1,36 @@
+import os
+import csv
+from django.core.management.base import BaseCommand
+
+from reports.report_builder import XLSXReportBuilder
+from reports.report_dataset import generate_chart_desc
+
+from reports.forms import GlobalForm
+
+
+class Command(BaseCommand):
+    help = "Generates dataset from the GMMP report"
+
+    def add_arguments(self, parser):
+        parser.add_argument("-w","--worksheets", nargs="+", help="Worksheets to generate dataset", required=False)
+        parser.add_argument("-c", "--chart", action="store_true", help="Generate names and descriptions for Wazimap-NG chart.")
```

Comment:
No need for this; we should always generate this file.. Unless there is a reason not to.